extensions/ql-vscode/src/codeql-cli/cli.ts

```diff
@@ -412,7 +420,9 @@ export class CodeQLCliServer implements Disposable {
         newError.stack += getErrorStack(err);
         throw newError;
       } finally {
-        void this.logger.log(Buffer.concat(stderrBuffers).toString("utf8"));
+        if (!silent) {
+          void this.logger.log(Buffer.concat(stderrBuffers).toString("utf8"));
+        }
```

Comment:
I've only wrapped the `logger.log` calls in the `if` statement. Some of the argument/buffer wrangling is possibly unnecessary if we don't log stuff, but I didn't want to mess with the internals of the command!

Reply:
Does this mean we are not logging even if there are errors?

Maybe that makes sense since `resolve queries` will be run in the background often and writing errors to the log will be very noisy. However, I wonder if users should be made aware of at least the first error they see?

Reply:
And, related, if you do start logging errors, there's no log entry of the `argstring` (I find this _extremely_ useful for debugging). So, the argstring should be emitted if errors are coming out.

And...now that I think about it, stderr in the case show _all_ log messages, so it's probably best to hide it. It would just be nice to ensure that errors are emitted along with the argstring.